hera_cal/lst_stack/calibration.py

```diff
@@ -335,6 +341,51 @@ def _lstbin_phase_calibration(
     return calibration_parameters, phase_gains
 
 
+def _lstbin_cross_pol_phase_calibration(
+    stack: LSTStack,
+    model: np.ndarray,
+):
+    """
+    This function performs cross-polarization phase calibration on LSTStack object by comparing each day to an input
+    reference model.
+    """
+    # Dictionaries for storing data used in calibration function
+    data = {}
+    abscal_model = {}
+    wgts = {}
+    baselines = []
+
+    # Loop through baselines and polarizations
+    for polidx, pol in enumerate(stack.pols):
+        pol1, pol2 = utils.split_pol(pol)
+
+        if pol1 == pol2:
+            continue
+
+        for apidx, (ant1, ant2) in enumerate(stack.antpairs):
+
+            # Move to the next blpol if there is not a model for the data or the entire baseline is flagged
+            if np.all(stack.flags[:, apidx, :, polidx]):
+                continue
+
+            # Get model, weights, and data for each baseline
+            wgts[(ant1, ant2, pol)] = stack.nsamples[:, apidx, :, polidx] * (
+                ~stack.flags[:, apidx, :, polidx]
+            ).astype(float)
+            abscal_model[(ant1, ant2, pol)] = model[apidx, :, polidx] * np.ones(
+                (len(stack.nights), 1)
+            )
+            data[(ant1, ant2, pol)] = stack.data[:, apidx, :, polidx]
+            baselines.append((ant1, ant2, pol))
+
+    # Perform cross-polarization phase calibration
+    delta = abscal.cross_pol_phase_cal(
+        model=abscal_model, data=data, model_bls=baselines, data_bls=baselines, wgts=wgts
```

Comment:
for clarity, perhaps explicitly set return_gains to False

Reply:
Changed this